.circleci/config.yml

```diff
@@ -87,7 +87,7 @@ jobs:
             .circleci/shared-helpers
       - *restore_npm_cache
       - node/install-npm:
-          version: "7"
+          version: '8'
```

Comment:
I've seen some problems in old packages because of a fix version '7' that we changed across some packages to '^7'
Do you think it causes any harm to change to '^8'? Or should we stick to a fixed version?

Reply:
I think in CircleCI `8` vs `^8` are the same thing. You can see that [in the build for this PR v `8.10.0` was installed](https://app.circleci.com/pipelines/github/Financial-Times/n-logger/979/workflows/8a08eb32-ad1f-441a-b3bb-f0245ff42988/jobs/3979?invite=true#step-104-47).

Reply:
Ignore for a minute until [this comment](https://github.com/Financial-Times/n-mask-logger/pull/71#discussion_r874545412) is resolved

Reply:
```suggestion
          version: '^8'
```